x/oracle/types/ballot_test.go

```diff
@@ -247,3 +247,34 @@ func TestPBStandardDeviation(t *testing.T) {
 		require.Equal(t, tc.standardDeviation, stdDev)
 	}
 }
+
+func TestPBStandardDeviation_Overflow(t *testing.T) {
```

Comment:
Keeping this out of `TestPBStandardDeviation` since that test set uses `float64` -> `sdk.Dec`, which we should probably refactor in a separate PR